build/signedbuild.yml

```diff
@@ -165,6 +168,29 @@ jobs:
       configuration: debug
       rerunFailedTests: true
 
+  - task: securedevelopmentteam.vss-secure-development-tools.build-task-uploadtotsa.TSAUpload@1
+    displayName: 'TSA upload to Codebase: accessibility-insights-windows_master Version: TsaV2'
+    inputs:
+      tsaVersion: TsaV2
+      codebase: NewOrUpdate
+      codeBaseName: 'accessibility-insights-windows_master'
+      notificationAlias: 'a11y-insights-eng@microsoft.com'
+      codeBaseAdmins: 'REDMOND\mreay;REDMOND\ahmohame'
+      instanceUrlForTsaV2: MSENG
+      projectNameMSENG: AzureDevOps
+      areaPath: 'AzureDevOps\VSTS\enSPIRe\Keros\Desktop'
+      iterationPath: 'AzureDevOps\OneVS'
+      uploadFxCop: true
+      uploadBinSkim: true
+      uploadCredScan: true
+      uploadAPIScan: false
+      uploadFortifySCA: false
+      uploadModernCop: false
+      uploadPoliCheck: false
```

Comment:
Aren't we required to run polycheck?

Reply:
We definitely need to be running policheck, although I don't know how we specify it via YML. Might be worth checking with the web team to see how they enabled it